src/Surfnet/StepupGateway/SecondFactorOnlyBundle/Service/Gateway/RespondService.php

```diff
@@ -105,6 +102,8 @@ public function respond(ResponseContext $responseContext)
         }
 
         $secondFactor = $this->secondFactorService->findByUuid($selectedSecondFactorUuid);
+        $this->responseValidator->validate($request, $secondFactor, $responseContext->getIdentityNameId());
+
```

Comment:
When do we use an extra empty line, is that personal preference?

Reply:
No hard requirements here. Only to not have a newline between the last closing parenthesis' of a PHP Class

```
    }

}
```
Yields:  `ERROR | [x] The closing brace for the class must go on the next line after the body`

In this case I wanted to emphasis the validation method.